AAC templates: Removing unwanted metadata (#52)
MicrosoftDocs/content-templates: templates/org/aac/aac-example-workload-content.md

> <!-- 

Reviewer: ```suggestion
<!--
```

Cost Optimization focuses on ways to reduce unnecessary expenses and improve operational efficiencies. For more information, see [Design review checklist for Cost Optimization](/azure/well-architected/cost-optimization/checklist).

> <!-- 

Reviewer: ```suggestion
<!--
```

> Explain Cost Optimization considerations. 
> Link to the pricing calculator (https://azure.microsoft.com/pricing/calculator), with this architecture prepopulated. 
> Include the major cost-driving components, a typical scale or throughput, and recommended SKUs. 

Reviewer: ```suggestion
Explain Cost Optimization considerations.
Link to the pricing calculator (https://azure.microsoft.com/pricing/calculator), with this architecture prepopulated.
Include the major cost-driving components, a typical scale or throughput, and recommended SKUs.
```

<!-- Explain key performance considerations, beyond the typical. -->

## Deploy this scenario

> <!-- This section is expected but optional if the contributors prefer to omit it.
> Link to a template or script that explains how to deploy the workload.
> Include source code, deployment code, and instructions for how to test the scenario. 

Reviewer: ```suggestion
<!--
This section is expected but optional if the contributors prefer to omit it.
Link to a GitHub-hosted template or script that explains how to deploy the workload.
The repo will contain all of the source code, deployment code, and instructions for how to try the scenario in the customer's own subscription.
```

Other contributors:

<!--

> This section is optional. 
> List contributors and technical reviewers. 

Reviewer: ```suggestion
This section is optional.
List contributors and technical reviewers.
```
